SampleUserControlLibrary/SampleUserControl.xaml.cs

```diff
@@ -104,6 +104,11 @@ public string SubscriptionEndpoint
             set;
         }
 
+        public void set_subscriptionPage_defaultEndpoint(string msg)
```

Comment:
1) Usually, C# function name follows PascalCase [1] rather than starting with lower case with underline as separator.
2) `msg` is obscure, change it to `endpoint` instead.
3) No `default` is necessary, details explained at other places.

[1] https://en.wikipedia.org/wiki/PascalCase